src/index.js

```diff
@@ -20,6 +20,14 @@ function IpfsAPI (host_or_multiaddr, port) {
     config.port = port || config.port
   }
 
+  // autoconfigure in browser
+  if (!config.host &&
+    window && window.location) {
+    var split = window.location.host.split(':')
```

Comment:
This looks like it works when the ipfsAPI is used from a `domain:port` url only. Do we want to use the url of the web application where the domain is being loaded for this?